telethon_examples/interactive_telegram_client.py

```diff
@@ -1,21 +1,20 @@
 import os
 from getpass import getpass
 
-from telethon import TelegramClient, ConnectionMode
-from telethon.errors import SessionPasswordNeededError
-from telethon.tl.types import (
-    UpdateShortChatMessage, UpdateShortMessage, PeerChat
-)
 from telethon.utils import get_display_name
 
+from telethon import ConnectionMode, TelegramClient
+from telethon.errors import SessionPasswordNeededError
+from telethon.tl.types import (PeerChat, UpdateShortChatMessage, UpdateShortMessage)
+
 
 def sprint(string, *args, **kwargs):
     """Safe Print (handle UnicodeEncodeErrors on some terminals)"""
     try:
         print(string, *args, **kwargs)
     except UnicodeEncodeError:
-        string = string.encode('utf-8', errors='ignore')\
-                       .decode('ascii', errors='ignore')
+        string = string.encode('utf-8', errors='ignore') \
+            .decode('ascii', errors='ignore')
```

Comment:
Why did this change (or why did you commit it)?

Reply:
It's autoformatting of pycharm..

Reply:
we seem to have different settings